packages/styling/src/interfaces/ISemanticTextColors.ts

```diff
@@ -64,17 +64,17 @@ export interface ISemanticTextColors {
   //// Invariants - slots that rarely change color theme-to-theme because the color has meaning
 
   /**
-   * The default color of text indicating error on bodyBackground.
+   * The default color of error text on bodyBackground.
    */
   errorText: string;
 
   /**
-   * The color of text indicating warning on errorBackground, warningBackground, blockingBackground, or successBackground.
+   * The default color of warning text on errorBackground, warningBackground, blockingBackground, or successBackground.
```

Comment:
same comment as before here, this isn't the color of warning text, it's just the default color of text altogether on these backgrounds

Reply:
Ah I thought it was consistent with the other three text colors - like errorText is the color of error text so I thought warningText is the color of warning text too...

Reply:
Yeah it's confusing.
1) errorText is just the dark red we use for normal error text on normal bodyBackground. The name is old and predates Fabric.
2) warningText is new, and designed to work as the default color of text on top of a variety of backgrounds. Main scenario is message bar, but we use it in lists too. The background color indicates whether the message is a warning, error, or merely informational.